kong/clustering/rpc/socket.lua

```diff
@@ -68,6 +68,11 @@ function _M._dispatch(premature, self, cb, payload)
   if not res then
     ngx_log(ngx_WARN, "[rpc] RPC callback failed: ", err)
 
+    -- notification has no response
+    if not payload.id then
+      return
+    end
```

Comment:
```suggestion
    -- notification has no response
    if not payload.id then
      ngx_log(ngx_DEBUG, "[rpc] notification has no response")
      return
    end
```

I think here we still need this debug log, allowing us to know whether we need to reply

Reply:
Sure.